tests/Services/ServiceTestCase.php

```diff
@@ -31,6 +31,16 @@ public function setUp(): void
         $this->service = $client->{$this->serviceKey};
     }
 
+    /**
+     * Helps you generate service quickly
+     */
+    public function createService($serviceKey)
+    {
+        $client = $this->createClient();
+
+        return $client->{$serviceKey};
+    }
+
```

Comment:
Adding helper method to quickly generate services when needed, in my case I needed `GamService` in `AdSlotService`